pages/Category.vue

```diff
@@ -46,26 +51,26 @@
             </SfSelectOption>
           </SfSelect>
           <SfButton
-            class="sf-button--text navbar__filters-button sort-by__button mobile-only"
+            class="sf-button--text navbar__button mobile-only"
             @click="$refs.SortBy.toggle()"
           >
             {{ $t('Sort By') }}
             <ASortIcon />
           </SfButton>
         </div>
+
         <div class="navbar__counter">
-          <span class="navbar__label desktop-only">
+          <span class="label desktop-only">
             {{ $t("Products found") }}:
           </span>
           <strong class="desktop-only">{{ getCategoryProductsTotal }}</strong>
-          <span class="navbar__label mobile-only">
+          <span class="count mobile-only">
```

Comment:
maybe we could add a space here to make it easy to differentiate the mobile and desktop UI. maybe even annotate them with comments?

Reply:
Ok